torch/csrc/jit/ir.cpp

```diff
@@ -531,4 +531,29 @@ void PythonOp::cloneFrom(Node * other_) {
   }
 }
 
+void renameInputsAndOutputs(const std::shared_ptr<Graph>& graph,
```

Comment:
Why do we need a rename function? Why not move the logic into NameInputsAndOutputs

Reply:
just in case I would have had to mess around with the includes of export.h

However I'll change it as you suggest (unless it's rendered moot by @apaszke's suggestion below)